### ISSUE TITLE
Install on windows

### ISSUE THREAD

**Issue by lucas-garcia** [zoomhub/node-deepzoomtools#8]
Does the library work on Windows? I'm hitting errors because of the postinstall command `_coffee -c lib`
What does this command do?

Update: Understood that is the command that compiles the coffee script, but still don't know why not working on windows

**Comment by gasi** [comment]
Thanks for reporting this. I am currently not maintaining this library but I welcome you to take a look. What is the error you are getting? `_coffee` is the Streamline.js compiler for CoffeeScript: https://github.com/Sage/streamlinejs I don’t have access to a Windows machine.

**Comment by aseemk** [comment]
What error message(s) are you getting, @lucas-garcia?

Can you also paste the output of `node --version`, `npm --version`, and `npm ls`?

**Comment by aseemk** [comment]
I'll fix this by compiling on `prepublish` rather than `postinstall` (as has generally become the preferred compiling method in the node/npm world, for better or worse).

@gasi: would you mind adding me to the npm module? `npm owner add aseemk deepzoomtools`. Thanks!

**Comment by gasi** [comment]
Done :+1:

**Comment by aseemk** [comment]
Thanks @gasi.

@lucas-garcia: try v0.0.7 from npm now.

**Comment by lucas-garcia** [comment]
Hey @aseemk , I'm very sorry for the very late answer.

I am working on a Mac but I reported the Windows problem because some colleagues have Windows and it was not working. But it looks like now it's broken for me. I installed it and was getting `Cannot find module 'deepzoomtools'`. So then I manually run `npm run build` and only then it compiled the files. Any idea what it could be?

Thanks!

**Comment by aseemk** [comment]
@lucas-garcia: I'm guessing you haven't updated to v0.0.7. Try that, it should fix it.

**Comment by lucas-garcia** [comment]
@aseemk Aren't you in v0.0.8? That's what I have here

**Comment by aseemk** [comment]
(Actually, update to v0.0.8 while you're at it. It contains an important bug fix.)

**Comment by aseemk** [comment]
You're right, I just installed a fresh 0.0.8 and am not seeing the compiled files (though the source maps are there). Will investigate and fix. Sorry about that!

**Comment by lucas-garcia** [comment]
No problem, thanks you for investigating it!
